src/app/models/proptypes.js

```diff
@@ -40,6 +40,11 @@ export const imagePropTypes = {
   }).isRequired,
 };
 
+export const linkPropTypes = shape({
```

Comment:
It would be fine to keep these propTypes in the Footer. They aren't used by other components and are trivial.